reference/exercise-concepts/bank-account.md

```diff
@@ -54,7 +54,7 @@ end
 
 ### GenServer
 
-[by @aaroncm (adapted)](https://exercism.io/tracks/elixir/exercises/bank-account/solutions/961596ab0cce445aa918452137feae1b)
+by @aaroncm (adapted)
```

Comment:
I couldn't find any bank account solutions from a person with that nickname

Reply:
Maybe they changed their handle...

Reply:
Jeremy made the old links work now and looks like this solution really doesn't exist anymore so removing the link is a good idea I guess.